packages/nulp_elite/src/components/CircularProgressWithLabel.js

```diff
@@ -5,8 +5,7 @@ import Box from "@mui/material/Box";
 
 const CircularProgressWithLabel = ({ received, total }) => {
   // Calculate the progress percentage
-  const progress = (received / total) * 100;
-
+  const progress = 50;
```

Comment:
We should calculate this instead of hardcoded number

Reply:
As per UI, we are showing a half circle if we calculate it based on value it will not match the UI design